### PR TITLE
Delete user

### PR DESCRIPTION
**Pull request by boeckhoff**

### CODE REVIEW

File: users/views.py [hunk 1]
```diff
@@ -22,3 +23,10 @@ def register(self, request):
         serializer.save()
         return Response(serializer.data)
 
+    @list_route(methods=['post'], permission_classes=[AllowAny])
```
Comment (nikriek) [inline users/views.py]: Please create a new view. You don't need the decorator as we are Not using viewsets anymore. 

Edit: I am wrong, we are using viewsets. ;) Thats why you don't need to add another url route

**Review by nikriek — CHANGES_REQUESTED** [review]

File: users/urls.py [hunk 2]
```diff
@@ -11,4 +11,5 @@
     url(r'^auth/login$', obtain_jwt_token, name='auth-jwt-obtain'),
     url(r'^auth/refresh$', refresh_jwt_token, name='auth-jwt-refresh'),
     url(r'^auth/verify$', verify_jwt_token, name='auth-jwt-verify'),
+    url(r'^auth/delete$', delete_jwt_token, name='delete-jwt-verify'),
```
Comment (nikriek) [inline users/urls.py]: You are Not deleting authentication but the user. Name the endpoint sonerhing like `/me`

Comment (nicoknoll) [reply users/urls.py]: Just use the DEL method at /users/me endpoint

Comment (nikriek) [reply users/urls.py]: Jep 👍

File: users/views.py (hunk 1 above)
Comment (nicoknoll) [reply users/views.py]: and don't use post but DELETE

File: users/urls.py (hunk 2 above)
Comment (nikriek) [reply users/urls.py]: It's inserted automatically with viewsets

File: users/views.py (hunk 1 above)
Comment (nikriek) [reply users/views.py]: And remove permission_classes ;)

File: users/urls.py (hunk 2 above)
Comment (nicoknoll) [reply users/urls.py]: rest-presentation-9-638

**Comment by boeckhoff** [comment]
tests will follow

**Pull request merged**